src/mailing/sender.js

```diff
@@ -43,5 +45,34 @@ async function sendTestEmail(to, username) {
   });
 }
 
-// Export the transport and the sendTestEmail function
-module.exports = { sendTestEmail };
+async function sendForgotPasswordEmail(toEmail, resetToken) {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: process.env.EMAIL_USER,
+      pass: process.env.EMAIL_PASSWORD,
+    },
+  });
+
+  // Load and render the HTML template
+  const htmlTemplate = await ejs.renderFile(
+    path.join(__dirname, './templates/forgot_password/html.ejs'),
+    { resetLink: `http://localhost:8000/api/v1/reset-password/${resetToken}` }
```

Comment:
Likely this URL will not be under `/api/v1` since it's not an API endpoint but rather a EJS/HTML page for the user to see and where they can input their resetted password.